internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quickstart
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+func (opts *Opts) askConfirmConfigQuestion() error {
+	if opts.Confirm {
+		return nil
+	}
+
+	diskSize := uint64(0)
+	if opts.newCluster().DiskSizeGB != nil {
+		diskSize = uint64(*opts.newCluster().DiskSizeGB)
+	}
```

Comment:
M0 comes in as nil, so setting to 0 in those instances

Reply:
M0 is 0.5GiB

Reply:
Cool, I'll update so the text shows that in this instance, thanks!